modules/ipc/src/zenoh/liveliness.cpp

```diff
@@ -208,9 +210,12 @@ void EndpointDiscovery::createLivelinessSubscriber() {
     }
   };
 
+  ::zenoh::ZResult result{};
   liveliness_subscriber_ =
       std::make_unique<::zenoh::Subscriber<void>>(session_->zenoh_session.liveliness_declare_subscriber(
-          keyexpr, std::move(liveliness_callback), ::zenoh::closures::none));
+          keyexpr, std::move(liveliness_callback), ::zenoh::closures::none, { .history = true }, &result));
```

Comment:
This works great! while you are at it, would you mind removing the code in the constructor that calls `getListOfEndpoints`? As it is no longer needed with this change